docs/docs/20-quickstart.mdx

```diff
@@ -25,7 +25,14 @@ This guide presents a basic introduction to Kargo. Together, we will:
     * kind: v0.17.0
     * k3d: v5.4.9
 * [Helm](https://helm.sh/docs/): These instructions were tested with v3.11.2.
-* The `kargo` CLI: Download it from [the releases page](https://github.com/akuity/kargo/releases/latest).
+* The `kargo` CLI: To install it, follow these steps.
+
+    1. Visit [the releases page](https://github.com/akuity/kargo/releases/latest) of the Kargo GitHub repository.
+    2. Download the appropriate release package for your operating system (e.g., `kargo-linux-amd64` for Linux).
+    3. Extract the contents of the downloaded package to a directory of your choice.
+    4. Add the directory containing the `kargo` executable to your system's `PATH` environment variable.
```

Comment:
Addressing @krancour's comment. Nothing to extract, instead instruct to move binary to a `bin` directory
```suggestion
    3. Move the binary to a directory of your choice (e.g. for Linux, consider `/usr/local/bin/`).
    4. Add the directory containing the `kargo` executable to your system's `PATH` environment variable.
```

Reply:
This suggested change didn't get merged.